### PR TITLE
Solving #4073

### PR DESCRIPTION
**Pull request by j0rt3g4**
formatted as GitHub.

#4073

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/policy-csp-authentication.md [hunk 1]
```diff
@@ -19,70 +19,43 @@ manager: dansimp
...
-    <a href="#authentication-preferredaadtenantdomainname">Authentication/PreferredAadTenantDomainName</a>
-  </dd>
-</dl>
 
 
+[Authentication/AllowAadPasswordReset](#Authentication-AllowAadPasswordReset)
+
+[Authentication/AllowEAPCertSSO](#Authentication-AllowEAPCertSSO)
+
+[Authentication/AllowFastReconnect](#Authentication-AllowFastReconnect )
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-authentication.md]:
```suggestion
[Authentication/AllowFastReconnect](#Authentication-AllowFastReconnect)
```

**Comment by mypil** [comment]
@ManikaDhiman - PR has been copy edited and is ready for final review. This is based on issue #4073 

Thank you.

**Comment by ManikaDhiman** [comment]
@j0rt3g4 We use code to parse the ```/mdm/policy*.md``` files, which is based on the existing syntax style. If you change the syntax style from HTML to MD, it breaks the automation code. Additionally, in the updated syntax, Scope is added as separate H3s, when it should be part of the given policy section itselft. The original H2s are also updated, which alters the "In this article" list. For example, we want Line#508 to appear in the "In this article" list, which will not appear now. Request you to please undo all the syntax changes. Thanks!

**Comment by j0rt3g4** [comment]
@ManikaDhiman  Will do

**Comment by j0rt3g4** [comment]
@mani

**Pull request closed without merging**